de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/actions/FollowModeAction.java

```diff
@@ -37,21 +33,17 @@ public void sessionEnded(ISarosSession oldSarosSession, SessionEndReason reason)
 
           ThreadUtils.runSafeAsync(
```

Comment:
I do not get the reason why we have to run a thread to set those variables. You could either dispatch this to the Swing Thread / EDT or just set the variables. See also line 24, same issue.

Reply:
I just modified the existing code. Removed.